xbmc/pvr/epg/EpgContainer.cpp

```diff
@@ -892,13 +892,13 @@ void CPVREpgContainer::SetHasPendingUpdates(bool bHasPendingUpdates /* = true */
 void CPVREpgContainer::UpdateRequest(int iClientID, int iUniqueChannelID)
 {
   std::unique_lock<CCriticalSection> lock(m_updateRequestsLock);
-  m_updateRequests.emplace_back(CEpgUpdateRequest(iClientID, iUniqueChannelID));
+  m_updateRequests.emplace_back(iClientID, iUniqueChannelID);
```

Comment:
Not sure this change is correct. Is really the right method called here still? `CEpgUpdateRequest` is not a `std::pair`, as far as I remember. And even though it is, I would like to stick with the real type name here.